nifi-api/src/main/java/org/apache/nifi/flowfile/FlowFile.java

```diff
@@ -108,7 +108,7 @@
      */
     Map<String, String> getAttributes();
 
-    public static class KeyValidator {
```

Comment:
Give that this is in the nifi-api I don't believe this change is ok outside of major nifi release boundary (nifi 2.0).  The only way this change would be ok is if having this class public and static was unintentional/a bug and that we are sure someone else should not be reasonably using this. Do you think that is the case?

Reply:
I agree. This was the IDE being aggressive about code cleanliness because any class defined in an `interface` is `public static` already. I can revert this change to make it explicitly apparent to callers, but there is no functional change here. See https://stackoverflow.com/a/7486111/70465.